### ISSUE TITLE
Convert this app to TypeScript

### ISSUE THREAD

**Issue by max-programming** [drkPrince/tweet-to-image#4]
Hey there! Really loved the concept and the app is so professional!

I thought how about we convert it to use TypeScript? As it's a really great app and TS helps a lot in catching bugs.

If you want, assign it to me and I will make a PR for that 👍

**Comment by drkPrince** [comment]
Thank You. You can definitely add TS. But I doubt it will be useful. It's not a very complex app.

**Comment by max-programming** [comment]
Yes it's not a too complex app but TS will still help. As Typescript provides awesome intellisense in IDE.

So errors can be discovered very easily. Assign this issue to me and I'll work on it 😁.

> This app is built using Next.js and it can easily be made a PWA too. I'll also work on that after Typescript if you want to
